code/game/gamemodes/cult/runes.dm

```diff
@@ -210,6 +210,8 @@ var/list/sacrificed = list()
 
 /obj/effect/rune/proc/seer()
 	if(usr.loc==src.loc)
+		// No. Bad cultcode. Bad.
+		var/mob/living/carbon/human/H = usr
```

Comment:
I might just be stupid, but wouldn't you wanna replace the rest of these `usr`'s with `H`?